app/sidekiq/lighthouse/evidence_submission_document_upload_polling_job.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
```

Comment:
Make sure you add all of these new files to CODEOWNERS

Reply:
Updated it :)